classes/Http/Controller/Reviewer/SpeakersController.php

```diff
@@ -13,7 +13,17 @@ class SpeakersController extends BaseController
     public function indexAction(Request $req)
     {
         $search = $req->get('search');
-        $speakers = User::search($search)->get()->toArray();
+        
+        if (!empty($req->get('order_by'))) {
+            $order_by = $req->get('order_by');
```

Comment:
Use camel-case notation for PHP variable names to be consistent with the rest of the codebase.

Reply:
Changed.